spec/requests/relationship_operations_spec.rb

```diff
@@ -291,29 +291,33 @@
     context 'authorized for remove_to_many_relationship' do
       context 'not limited by policy scopes' do
         before do
-          allow_operations('remove_to_many_relationship', [
-            [article, comments_to_remove.first, :comments],
-            [article, comments_to_remove.second, :comments]
-          ])
+          allow_operation('remove_to_many_relationship', article, kind_of(Array), "comments")
         end
 
         it { is_expected.to be_successful }
       end
 
       context 'limited by policy scope on comments' do
-        let(:comments_scope) { Comment.none }
         before do
-          allow_operation('remove_to_many_relationship', article, kind_of(Comment), :comments)
+          allow_any_instance_of(ArticlePolicy).to receive(
+            :remove_from_comments?).and_return(true)
```

Comment:
It would be great if we didn't have to worry about policies in here at all, and could create this spec in terms of only the operations stubbing instead.

Would it be possible to restructure this spec to look something like this?

```rb
context 'limited by policy scope on comments' do
  let(:comments_scope) { Comment.none }
  before do
    allow_operation('remove_to_many_relationship', article, [], :comments)
  end

  it { is_expected.to be_succesful }
end
```

This would also ensure that the `remove_to_many_relationship` is indeed called with an empty array instead of any comments.